docs/source/scilla-in-depth.rst

```diff
@@ -435,6 +435,27 @@ same name must have the same entry names and types.
    at any point during execution, but the event will not be visible on
    the blockchain before the transition has completed.
 
+Finally, a contract can raise errors by throwing exceptions. Any error
```

Comment:
imho, this paragraph deserves its own section ("Error handling"), even if that will be just a stub at the moment